docker/README.md

```diff
@@ -70,3 +70,26 @@ After build finished, run the container:
 ```
 docker run --rm -it lightgbm
 ```
+
+## Running the R-package Сontainer
+
+Build the container based on the [`verse` Rocker image](https://www.rocker-project.org/images/), for R users:
+
+```
+mkdir lightgbm-docker
+cd lightgbm-docker
+wget https://raw.githubusercontent.com/Microsoft/LightGBM/master/docker/dockerfile-r
+docker build -t lightgbm-r -f dockerfile-python .
```

Comment:
you accidentally left `dockerfile-python` here, I think you meant `dockerfile-r`